simapp/app.go

```diff
@@ -458,15 +458,15 @@ func NewSimApp(
 		distrtypes.ModuleName,
 		slashingtypes.ModuleName,
 		evidencetypes.ModuleName,
-		poa.ModuleName,
 		stakingtypes.ModuleName,
+		poa.ModuleName, // after staking
 		genutiltypes.ModuleName,
 		authz.ModuleName,
 	)
 	app.ModuleManager.SetOrderEndBlockers(
 		crisistypes.ModuleName,
 		govtypes.ModuleName,
-		poa.ModuleName,
+		poa.ModuleName, // Before staking
```

Comment:
TODO if this fixes CI need to note (since it will process in staking before clearing out all in the next round w/ POA)